pom.xml

```diff
@@ -6034,7 +6034,8 @@
                     <!-- Exclusions are tolerated for nightly "date-based" releases only -->
                     <excludes>
                       <exclude>org.nuxeo:nuxeo-ftest:*:zip</exclude>
-                      <!-- <exclude>org.nuxeo.connect:nuxeo-connect-client</exclude> -->
+                      <!-- TODO NXBT-1765 Disable the next line after the release of Nuxeo Connect Client 1.5.1 -->
```

Comment:
Thanks for your explanation @jcarsique.

Ticket [NXP-22581](https://jira.nuxeo.com/browse/NXP-22581) is now created for aligning on `org.nuxeo.connect:nuxeo-connect-client` 1.5.1 release. The line will be removed in PR https://github.com/nuxeo/nuxeo/pull/891